### ISSUE TITLE
<a> tag's href value gets html encoded.

### ISSUE THREAD

**Issue by kanup** [zzzprojects/html-agility-pack#44]
In version 1.5.0, I could do 
var url = node.SelectSingleNode(".//h3[@class='xxx-item-title']/a").Attributes["href"].Value;
new HtmlWeb().Load(url);

In version 1.5.1 this doesn't work because the ampersands (&) in the url gets decode into `&amp;`

**Comment by JonathanMagnan** [comment]
Hello @kanup ,

You are right.

In the version 1.5.0, we fixed the issue about the Value property that doesn't DeEntitize the value. However, some people like in the issue #32 reported us that our fix was breaking their code. So we reverted our change and added a new property `DeEntitizeValue` which should return the value expected in your case.

Currently, it's debate about if we should stay backward compatible or not until the version 2.x is released.

We understand the `Value` property should automatically DeEntitize the value but since the library has always worked like this, we currently prefer not fixing it since it may break a lot of projects already using this library.

Let me know if the `DeEntitizeValue` property is working as expected.

Best Regards,

Jonathan
